code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -254,6 +255,17 @@
 		. = TRUE
 	..()
 
+/datum/reagent/consumable/soymilk/overdose_process(mob/living/M, delta_time)
+	metabolization_rate = 1 * REAGENTS_METABOLISM
+	if(ishuman(M))
+		if(DT_PROB(5, delta_time))
+			var/static/list/filth = list("silicons", "ethereals", "plasmamen", "moths", "lizards") //Writing the OD speeches was fun, good times.
+			M.say(pick("Opening those soy bottles is hard, i think my strength is waning...", "Honestly, i think [pick (filth)] deserve rights.", "Did you see my profile? I got like 20 thousand likes!", "Did you see the NT announcement where they spoke out about violence against [pick(filth)]? I really should support them more!", "I wish we could get rid of security so there wouldn't be any more violence.", "This place needs more exposed brick walls, maybe even a barcade!"))
+		if(DT_PROB(3.5, delta_time))
+			M.visible_message(pick("<b>[M]</b> looks a bit pale...", "<b>[M]</b>'s eyes lose focus.", "<b>[M]</b> slouches over.", "<b>[M]</b>'s mouth hangs a bit open."))
```

Comment:
```suggestion
			M.visible_message("<b>[M]</b>[pick(" looks a bit pale...", "'s eyes lose focus.", " slouches over.", "'s mouth hangs a bit open.")]")
```

Reply:
I would prefer if this was an emote call but I am not too sure on adding all of these as emotes.

Reply:
Oh that's neat, added it, commit soon.